.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,8 @@
-Fixes #ISSUE_NUMBER
+### Description
+<!-- What did you change and why was it needed? -->
+
+### Issue
+<!-- Link to Issue ticket or RFP -->

```

Comment:
Do you think #ISSUE_NUMBER might be easier here? GH automatically links it for you, i.e. `#81991` will become #81991

```suggestion
#ISSUE_NUMBER
```

Reply:
I figure github links it either ways, and most folks know that, so they can use either option here.  

The main thing I was hoping to emphasize with this comment is that if folks submitted an RFP then they should link those here.